packages/sources/superstate/src/config/index.ts

```diff
@@ -0,0 +1,21 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig(
+  {
+    API_ENDPOINT: {
+      description: 'An API endpoint for Superstate',
+      type: 'string',
+      default: 'https://api.superstate.co/v1',
+    },
+    LOOKBACK_DAYS: {
```

Comment:
@alecgard saw a previous comment on this env var and Karen explained it well, only thing I have to add is that the recommended lookback in the ticket was suggested to be 7 days, and the tradeoff with higher lookback is increased "padding" on ensuring _any_ response (unlikely edge case) vs increased DP response payload size in the normal case.
There shouldn't ever be a case where there is no update beyond 1 week (even Christmas + New year or Thanksgiving periods in the US are not expected to be 7 consecutive bank holiday days).
Further discussion in the ticket comments.